daemon/daemon.go

```diff
@@ -250,15 +250,9 @@ func (d *Daemon) Dying() <-chan struct{} {
 	return d.tomb.Dying()
 }
 
-func (d *Daemon) auther(r *http.Request) (store.Authenticator, error) {
+func (d *Daemon) auther(user *auth.UserState) (store.Authenticator, error) {
```

Comment:
The rework in #1521 pushes me further toward the anonymous UserState approach, since we no longer have a State to sensibly play with.